### ISSUE TITLE
Low-res/blurred images on Safari 

### ISSUE THREAD

**Issue by ertdfgcvb** [fxhash/fxhash-website#457]
Images won't load on Safari, the blurred preview stays in place (site wide).
Works as expected on FF and Chrome.

Tested on:
Safari Version 15.6.1
MacOS Version 10.15.7

<img width="1149" alt="Screenshot 2022-10-30 at 16 26 14" src="https://user-images.githubusercontent.com/864073/198887142-8a9e701d-e966-44fc-8edd-185a0ab3b356.png">

**Comment by GemN** [comment]
Hello,
Does this error still happen? I could not reproducing on my side.

**Comment by ertdfgcvb** [comment]
Yes. I tried deleting the cache / history but it stays the same. !
In the following screenshot you can see that the low-rez `data:image` images have been loaded but what I suppose must be the correct ones are all not reachable (in red). 
The error message is simply: 
An error occurred trying to load the resource. 

Maybe an issue in the webp images? 

<img width="1166" alt="Screenshot 2022-11-02 at 11 54 13" src="https://user-images.githubusercontent.com/864073/199472098-c5a57d99-de2d-4f1f-8750-6b8d9c80a172.png">

**Comment by GemN** [comment]
On my side: 
- Safari Version 15.6.1
- MacOS 11.6.8

I don't think it's a mac os version issue but at worst we can think about it.

1. Do you happens to browse behind a proxy / company firewall? 
2. Can you access a webp image directly ? (for ex: https://media.fxhash.xyz/w_768/QmaFLKK2N16xvKUkNxzUxkB1iSzqFttf9xwXLVmP3acgmf )
3. Does it work on other browsers ? (ex: chrome, firefox)

**Comment by ertdfgcvb** [comment]
> Works as expected on FF and Chrome.

I also disabled all proxies… so the files are definitely reachable. 
When i click the direct link the file gets downloaded, like a generic blob. Could it be that the MIME type for webp is not set correctly on the server / header?

**Comment by GemN** [comment]
Does the webp display correctly on this page ? https://developers.google.com/speed/webp/gallery
I did a little bit of research, if it doesn't work then it sounds like your Mac version (Catalina) has trouble displaying webp images (see here: https://www.reddit.com/r/MacOS/comments/pw1b0v/safari_15_does_not_show_some_webp_images/).

If that's the case, until we push an image fallback fix - I'm suggesting you to either use other browsers or update your MacOS version (if you can).

**Comment by ertdfgcvb** [comment]
It seems indeed to be a WebP issue… Weird that it is OS dependent and not browser dependent. 
Again: the page you linked works as expected on FF & Chrome…
 
<img width="1462" alt="Screenshot 2022-11-04 at 00 01 07" src="https://user-images.githubusercontent.com/864073/199851163-b2e628e0-9fa5-4771-8f59-a865f2552c4d.png">
